sql/core/src/test/resources/tpcds-plan-stability/approved-plans-modified/q7.sf100/explain.txt

```diff
@@ -10,15 +10,15 @@ TakeOrderedAndProject (34)
                :     :- * Project (17)
                :     :  +- * BroadcastHashJoin Inner BuildRight (16)
                :     :     :- * Project (10)
-               :     :     :  +- * BroadcastHashJoin Inner BuildLeft (9)
-               :     :     :     :- BroadcastExchange (5)
-               :     :     :     :  +- * Project (4)
-               :     :     :     :     +- * Filter (3)
-               :     :     :     :        +- * ColumnarToRow (2)
-               :     :     :     :           +- Scan parquet default.date_dim (1)
-               :     :     :     +- * Filter (8)
-               :     :     :        +- * ColumnarToRow (7)
-               :     :     :           +- Scan parquet default.store_sales (6)
+               :     :     :  +- * BroadcastHashJoin Inner BuildRight (9)
```

Comment:
I'm skimming the changes, and this seems non-trivial? but then again I am not so sure how to read these plans expertly enough to evaluate. The plan below starts by scanning a different table, for example.

We may just have to accept changes like this, if they're equivalent, to make sure they do not depend on implementation details of hash maps. But @gatorsmile @cloud-fan et al do these look like plausible equivalent plans for example?